Mathlib/Probability/Distributions/Gamma.lean

```diff
@@ -70,11 +70,18 @@ lemma lintegral_gammaPdf_of_nonpos {x a r : ℝ} (hx : x ≤ 0) :
     apply ae_of_all _ fun a (_ : a < _) ↦ by rw [if_neg (by linarith)]
 
 /-- The gamma pdf is measurable. -/
+@[measurability]
 lemma measurable_gammaPdfReal (a r : ℝ) : Measurable (gammaPdfReal a r) :=
   Measurable.ite measurableSet_Ici (((measurable_id'.pow_const _).const_mul _).mul
     (measurable_id'.const_mul _).neg.exp) measurable_const
 
-/-- the Gamma pdf is positive for all positive reals -/
+/-- The gamma Pdf is strongly measurable -/
+@[measurability]
+ lemma stronglyMeasurable_gammaPdfReal (a r : ℝ) :
+     StronglyMeasurable (gammaPdfReal a r) :=
+   (measurable_gammaPdfReal a r).stronglyMeasurable
+
+/-- The Gamma pdf is positive for all positive reals -/
```

Comment:
One docstring says `gamma Pdf`, the other `Gamma pdf`. Please make the capitalization consistent. (Also see other parts of the file.)